flow/layers/container_layer.cc

```diff
@@ -67,19 +80,72 @@ void ContainerLayer::PaintChildren(PaintContext& context) const {
 
 #if defined(OS_FUCHSIA)
 
+void ContainerLayer::CheckForChildLayerBelow(PrerollContext* context) {
+  // All ContainerLayers make the check in PrerollChildren.
+}
+
 void ContainerLayer::UpdateScene(SceneUpdateContext& context) {
   UpdateSceneChildren(context);
 }
 
 void ContainerLayer::UpdateSceneChildren(SceneUpdateContext& context) {
+  auto update_scene_layers = [&] {
+    // Paint all of the layers which need to be drawn into the container.
+    // These may be flattened down to a containing
```

Comment:
This comment seems incomplete, but I guess it already was before you got here :)

I think it meant to say "these may be flattened down into a containing Scenic Frame"